### ISSUE TITLE
revertAnimation only change background but not image and spinningbar color

### ISSUE THREAD

**Issue by ClarkXP** [leandroBorgesFerreira/LoadingButtonAndroid#29]
I using CircularProgressImageButton. Then when I call revertAnimation with callback, only is changed the background, isn't changing ImageDrawable and SpinningBarColor.

```java
btnPersonFollow.revertAnimation(new OnAnimationEndListener() {
            @Override
            public void onAnimationEnd() {
                boolean followingOrPending = Arrays.asList(FollowStatus.FollowStatusAccepted, FollowStatus.FollowStatusPending, FollowStatus.FollowStatusRejected).contains(getPerson().followingStatus);
                if (followingOrPending) {
                    btnPersonFollow.setBackgroundResource(R.drawable.bg_button_following);
                    btnPersonFollow.setImageResource(R.drawable.person_check);
                    btnPersonFollow.setSpinningBarColor(ResourcesCompat.getColor(viewHolder.itemView.getResources(),R.color.suda_blue_medium,null));

                } else {
                    btnPersonFollow.setBackgroundResource(R.drawable.bg_button_following_outline);
                    btnPersonFollow.setImageResource(R.drawable.person_add);
                    btnPersonFollow.setSpinningBarColor(ResourcesCompat.getColor(viewHolder.itemView.getResources(),R.color.suda_cyan,null));
                }
            }
        });
```

**Comment by leandroBorgesFerreira** [comment]
I will take a look at this @ClarkXP.

**Comment by leandroBorgesFerreira** [comment]
There was a bug when changing the image back was you wanted. I got fixed in the 1.8.4 version. 

But I didn't understand about spinning bar color. Why do you need to change it when the button is morphing back? No one is going to see it, right?

**Comment by ClarkXP** [comment]
I change the spinning bar color for the next excecution, is a button with two states.

**Comment by ClarkXP** [comment]
I had see that I can change the color if mAnimatedDrawable is nullified in end animation listener of revertAnimation()

**Comment by leandroBorgesFerreira** [comment]
Hello @ClarkXP. Sorry for taking so long to answer your request. I believe it got corrected in the version 1.10.0. Now you can change correctly the colour of the spinning bar. To change the icon, just follow the example in the MainActivity.

Like this: 

```
animatedButton.revertAnimation {
            animatedButton.setImageBitmap(
BitmapFactory.decodeResource(resources, R.drawable.ic_alarm_on_white_48dp))
}
```
